actionview/test/template/url_helper_test.rb

```diff
@@ -673,11 +669,6 @@ def test_url_for_shows_only_path
     assert_equal '/url_helper_controller_test/url_helper/show_url_for', @response.body
   end
 
-  def test_overridden_url_for_shows_only_path
-    get :show_overridden_url_for
-    assert_equal '/url_helper_controller_test/url_helper/show_url_for', @response.body
-  end
```

Comment:
So this one was broken too. I know it is usually a bad sign to delete a test, but in this case I believe it wasn't testing anything. The test just above test the exact same thing, except it passes a hash directly and this one is merging into the `params`.

I checked, there is no other parameters that were passed to `url_for`.